### ISSUE TITLE
Add Flatpak or Snap

### ISSUE THREAD

**Issue by baimafeima** [proxysh/Safejumper-for-Desktop#5]
Could you add Flatpak? This would make adoption much easier across many operating systems. 
http://flatpak.org/
The chance that your application lands in the repositories of any operating system is pretty low and compiling it oneself is too much to ask from an average end user. Binaries such as .deb or .rpm files have a limited use case and as I am moving away from both Debian/Ubuntu and Fedora, the only feasable way to run Safejumper is via Flatpak. This would also decrease maintanence work for you as you could replace .rpm entirely. Fedora has already adopted Flatpak as a cross-platform packaging format.

**Comment by threemonkeys** [comment]
@jpwhiting What do you think about that?

**Comment by proxysh** [comment]
Sounds like a good idea.

@jpwhiting What do you think about that?

**Comment by jpwhiting** [comment]
Yes, flatpak could be a way to simplify the linux releases we do. I need to check what distributions are supported with flatpak (i.e. does it support the older Ubuntu releases Safejumper supports or only newer releases of Ubuntu) and if so we should be good to go.

**Comment by Kabouik** [comment]
Flatpak would be great indeed, any news about this?

**Comment by proxysh** [comment]
We have other priorities at the moment, but we will not hesitate to take it into account soon.

**Comment by baimafeima** [comment]
An alternative would be making it available as snaps.

**Comment by proxysh** [comment]
We believe Flatpak would be a better option if we move forward with any option, what do you think?

**Comment by Kabouik** [comment]
I don't have experience with either Snaps or Flatpak, but for what it's worth, there's a [new distribution supporting snaps with full confinement since today: Solus](http://news.softpedia.com/news/solus-3-debuts-with-support-for-ubuntu-s-snaps-linux-4-12-budgie-desktop-10-4-517407.shtml?utm_content=buffer77a3c&utm_medium=social&utm_source=plus.google.com&utm_campaign=buffer). It's a very good distribution that's gaining a lot of popularity since a few months. Flatpak is supported as well but I heard it doesn't provide a great user experience (haven't tried myself)?

**Comment by proxysh** [comment]
It's a click-to-download solution so it should provide a good user experience, but we'll keep an eye on Solus.

**Comment by baimafeima** [comment]
How's the snapification coming along?

**Comment by proxysh** [comment]
It's not a priority at the moment, we're working on things like a better feedback system to report bugs. But your ticket is kept open for the future. :)

**Comment by baimafeima** [comment]
Just a note on the snap-flatpak issue: I think snaps are currently the more popular format and may be the better choice in the long run. As @Kabouik mentioned, the Solus operating system and the entire Ubuntu ecosystem are in favor of snaps (even though they support both), so this may be the best way to go forward. It would also allow for easy integration into any software center, whether it'll be Solus Software Center or Gnome Software Center.

**Comment by proxysh** [comment]
This is also related to https://github.com/proxysh/Safejumper-for-Desktop/issues/10#issuecomment-354948601
